packages/holocron-dev-server/src/webpack/plugins/holocron-webpack-plugin.js

```diff
@@ -50,24 +48,15 @@ export default class HolocronModulePlugin {
 
   apply(compiler) {
     compiler.hooks.compilation.tap(this.constructor.name, (compilation) => {
-      // eslint-disable-next-line default-case
-      switch (getWebpackVersion()) {
-        case 4: {
-          compilation.hooks.normalModuleLoader.tap(
-            this.constructor.name,
-            this.loaderHook.bind(this)
-          );
-          break;
-        }
-        case 5: {
-          const { webpack } = compiler;
-          const { NormalModule } = webpack;
-          NormalModule.getCompilationHooks(compilation).loader.tap(
-            this.constructor.name,
-            this.loaderHook.bind(this)
-          );
-          break;
-        }
+      if (typeof compiler.webpack !== 'undefined' && parseInt(compiler.webpack.version, 10) === 5) {
```

Comment:
@JAdshead  is worth it having this check considering we still have to use `getWebpackVersion()` in other places

Reply:
i think requiring webpack could lead to unexpected results within this context. when we move to 5 this will be a much cleaner solution.